examples/00-mapdl-examples/statically_indeterminate_reaction_force_analysis.py

```diff
@@ -0,0 +1,99 @@
+"""
+.. _ref_how_to_add_an_example_reference_key:
+
+Statically Indeterminate Reaction Force Analysis
+------------------------------------------------
+Taken from "The Strength of Materials" - Timoshenko, Part 1, 3rd Edition, Page 26, Problem 10
+
+Find the reaction forces at opposite, fixed, ends of a linking element 10 units long, with
+forces applied in the dimension of the element of 500 and 1000 at 4 and 7 units along the
+element.
+"""
+from ansys.mapdl.core import launch_mapdl
+
+# start MAPDL and enter the pre-processing routine
+mapdl = launch_mapdl()
+mapdl.clear()
+#
+mapdl.run("/VERIFY,VM1")
+mapdl.prep7()
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material and its type (a single material, with a linking-type
+# section and an enthalpy of 30e6).
```

Comment:
```suggestion
# section and a Young's modulus of 30e6).
```

Reply:
this was due to a mistake in the mp docstring.